session_0/fizz_buzz.rb

```diff
@@ -37,5 +37,27 @@
 # We will raise an `ArgumentError` exception to let the caller know that
 # their function arguments were incorrect.
 def fizz_buzz(n:, x: 3, y: 5)
```

Comment:
Please don't push session 0 and 1 again - using branches and pushing code correctly is part of learning how to use Git.

Can you remove them from the MR?

Reply:
Sorry everthing got messed up after i tried to update my repo :p